js/new_solution.test.js

```diff
@@ -1,10 +0,0 @@
-// const [function] = require('./[filename]);
```

Comment:
Did you actually mean to delete this file :smile: ?

Reply:
It was intentional. I will prepare script to create new files.